src/components/DelimitedList/DelimitedList.stories.js

```diff
@@ -3,46 +3,32 @@
  * @copyright IBM Security 2019
  */
 
-import { withA11y } from '@storybook/addon-a11y';
-import centered from '@storybook/addon-centered/react';
 import { array, boolean, text } from '@storybook/addon-knobs';
 import { storiesOf } from '@storybook/react';
 
+import { settings } from 'carbon-components';
 import React from 'react';
 
-import { settings } from 'carbon-components';
-import { components } from '../../../.storybook';
+import { components, withInfo } from '../../../.storybook';
 
 import { DelimitedList } from '../..';
 
-import props from './_mocks_';
-
-const { prefix } = settings;
+import mocks from './_mocks_';
 
 const { delimiter, truncate } = DelimitedList.defaultProps;
 
-const storyProps = () => ({
-  delimiter: text('Delimiter (`delimiter`)', delimiter),
-  items: array('Items (`items`)', props.items),
-  truncate: boolean('Truncate (`truncate`)', truncate),
+const props = () => ({
+  items: array('Items (items)', mocks.items),
+  delimiter: text('Delimiter (delimiter)', delimiter),
+  truncate: boolean('Truncate (truncate)', truncate),
 });
 
-storiesOf(components('DelimitedList'), module)
-  .addDecorator(withA11y)
-  .addDecorator(centered)
-  .add(
-    'Default',
-    () => (
-      <div className={`${prefix}--grid`}>
-        <div className={`${prefix}--row`}>
-          <div className={`${prefix}--col-sm-2`}>
-            <DelimitedList {...storyProps()} />
-          </div>
-        </div>
-      </div>
-    ),
-    {
-      info:
-        'The `DelimitedList` component truncates an array of items, separated by a delimiter, and includes the total number of items when hovering',
-    }
-  );
+storiesOf(components('DelimitedList'), module).add(
+  'Default',
+  () => (
+    <DelimitedList className={`${settings.prefix}--col-sm-2`} {...props()} />
```

Comment:
This example is now missing the row and grid wrapper classes. And the grid column class shouldn't be applied directly to the component. 🤔 

I think we should return this example to the way it was, so that we are demonstrating best practices.

Reply:
@jendowns Thanks - I think the extraneous grid classes, in general, distort the meaning of the component's purpose (to truncate once it's reached the appropriate width) and pollute the example output. I'll revisit to find a better overall solution